### PR TITLE
cloud: update the free quota of columnar storage

### PR DESCRIPTION
**Pull request by joey-yez**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/select-cluster-tier.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ TiDB Serverless offers two service plans to meet different user requirements. Wh
 The free cluster plan is ideal for those who are getting started with TiDB Serverless. It provides developers and small teams with the following essential features:
 
 - **No cost**: This plan is completely free, with no credit card required to get started.
-- **Storage**: Provides an initial storage capacity of 5 GiB.
+- **Storage**: Provides an initial row-based storage capacity of 5 GiB and an initial columnar storage capacity of 5 GiB.
```
Comment (Oreoxmt) [inline tidb-cloud/select-cluster-tier.md:30]:
```suggestion
- **Storage**: Provides an initial 5 GiB of row-based storage and 5 GiB of columnar storage.
```

File: tidb-cloud/serverless-faqs.md [hunk 2]
```diff
@@ -91,6 +121,20 @@ Once a cluster reaches its free quota or spending limit, the cluster immediately
 
 During the data import process of a TiDB Serverless cluster, RU consumption occurs only when the data is successfully imported, which leads to spikes in RU usage.
 
+### What costs are involved when using columnar storage in TiDB Serverless?
+
+The pricing for columnar storage in TiDB Serverless is similar to that for row-based storage. When you use columnar storage, an additional replica is created to store your data (without indexes). The synchronization of data from row-based to columnar storage does not incur extra charges.
```
Comment (qiancai) [inline tidb-cloud/serverless-faqs.md:126]:
```suggestion
The pricing for columnar storage in TiDB Serverless is similar to that for row-based storage. When you use columnar storage, an additional replica is created to store your data (without indexes). The replication of data from row-based to columnar storage does not incur extra charges.
```

File: tidb-cloud/serverless-faqs.md [hunk 3]
```diff
@@ -91,6 +121,20 @@ Once a cluster reaches its free quota or spending limit, the cluster immediately
...
+
+For detailed pricing information, see [TiDB Serverless pricing details](https://www.pingcap.com/tidb-serverless-pricing-details/).
+
+### Is using columnar storage more expensive?
+
+Columnar storage in TiDB Serverless incurs additional costs due to the extra replica, which requires more storage and resources for data synchronization. However, columnar storage becomes more cost-effective when running analytical queries.
```
Comment (qiancai) [inline tidb-cloud/serverless-faqs.md:132]:
```suggestion
Columnar storage in TiDB Serverless incurs additional costs due to the extra replica, which requires more storage and resources for data replication. However, columnar storage becomes more cost-effective when running analytical queries.
```

File: tidb-cloud/serverless-faqs.md [hunk 4]
```diff
@@ -36,6 +36,35 @@ We are actively working on expanding TiDB Serverless to other cloud platforms, i
...
+In these scenarios, columnar storage can significantly improve query performance and provide a seamless experience for mixed workloads in your system.
+
+### How to use columnar storage in TiDB Serverless?
+
+Columnar storage usage in TiDB Serverless is similar to that in TiFlash. You can enable columnar storage at both the table and database levels:
+
+- Table level: Assign a TiFlash replica to a table to enable columnar storage for that specific table.
+- Database level: Configure TiFlash replicas for all tables in a database to use columnar storage across the entire database.
+
+Once a TiFlash replica is set up for a table, TiDB automatically synchronizes the columnar storage with the row-based storage for that table. This ensures data consistency and optimizes performance for analytical queries.
```
Comment (qiancai) [inline tidb-cloud/serverless-faqs.md:64]:
```suggestion
Once a TiFlash replica is set up for a table, TiDB automatically replicates data from the row-based storage to the columnar storage for that table. This ensures data consistency and optimizes performance for analytical queries.
```

Comment (qiancai) [reply tidb-cloud/serverless-faqs.md:64]: synchronize 通常指的是双向的数据同步

**Review by qiancai — APPROVED** [review]
Rest LGTM

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**